pot.md

```diff
@@ -112,24 +112,25 @@ Pot Semantics
 
     syntax PotStep ::= "join" Wad
  // -----------------------------
-    rule <k> Pot . join WAD => call Vat . move MSGSENDER THIS ( CHI *Rat WAD ) ... </k>
+    rule <k> Pot . join PORTION => call Vat . move MSGSENDER THIS ( CHI *Rat PORTION ) ... </k>
          <this> THIS </this>
          <current-time> NOW </current-time>
          <msg-sender> MSGSENDER </msg-sender>
-         <pot-pies> ... MSGSENDER |-> ( MSGSENDER_PIE => MSGSENDER_PIE +Rat WAD ) ... </pot-pies>
-         <pot-pie> PIE => PIE +Rat WAD </pot-pie>
+         <pot-pies> ... MSGSENDER |-> ( MSGSENDER_PIE => MSGSENDER_PIE +Rat PORTION ) ... </pot-pies>
+         <pot-pie> PIE => PIE +Rat PORTION </pot-pie>
          <pot-chi> CHI </pot-chi>
          <pot-rho> RHO </pot-rho>
       requires NOW ==Int RHO
 
     syntax PotStep ::= "exit" Wad
  // -----------------------------
-    rule <k> Pot . exit WAD => call Vat . move THIS MSGSENDER ( CHI *Rat WAD ) ... </k>
+    rule <k> Pot . exit PORTION => call Vat . move THIS MSGSENDER ( CHI *Rat PORTION ) ... </k>
          <this> THIS </this>
          <msg-sender> MSGSENDER </msg-sender>
-         <pot-pies> ... MSGSENDER |-> ( MSGSENDER_PIE => MSGSENDER_PIE -Rat WAD ) ... </pot-pies>
-         <pot-pie> PIE => PIE -Rat WAD </pot-pie>
+         <pot-pies> ... MSGSENDER |-> ( MSGSENDER_PIE => MSGSENDER_PIE -Rat PORTION ) ... </pot-pies>
+         <pot-pie> PIE => PIE -Rat PORTION </pot-pie>
          <pot-chi> CHI </pot-chi>
+      requires MSGSENDER_PIE >=Int PORTION
```

Comment:
probably a style thing, but would it be more explicit if this line was `requires PIE -Rat Portion >=Int 0`?

Reply:
I fixed `>=Int` to `>=Rat`.

I guess I think of this check as "msg.sender has enough", not really an underflow check.  If you'd prefer the side-condition change I can do that.

Reply:
I thought of it the same way "msg.sender has enough" but since I was expecting "won't underflow" I thought the side-condition was more clear.  But that was just my initial bias. I actually feel that "msg.sender has enough" is more clear overall.  So I think we're good